cmd/endpoint.go

```diff
@@ -142,6 +153,14 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 	}, nil
 }
 
+// checkIP4 - helper for validating if the provided arg is an ip address.
+func checkIP4(ipAddress string) bool {
```

Comment:
https://golang.org/pkg/net/#ParseIP - there is already a library function to check for IP address - it will be better to use that than write another one ourself.